src/pages/Home/HomePage.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Grid, Pagination } from 'semantic-ui-react';
+
+import { useAuth } from '../../providers/Auth';
+import Header from '../Header';
+import SideBar from '../SideBar';
+import './Home.styles.css';
```

Comment:
There is a mix of `css` files and `styled-components`. It is not wrong because it works but I don't think it is a good practice. This component is small but as the codebase keeps growing it'll be hard to maintain.